src/main/java/duke/ListCommand.java

```diff
@@ -0,0 +1,22 @@
+package duke;
+
```

Comment:
Perhaps you can have a Javadoc comment here briefly describing the class?

Reply:
Alright thanks